doc/data/messages/p/possibly-used-before-assignment/good.py

```diff
@@ -0,0 +1,5 @@
+def func(value):
+    value = False
+    if value:
+        has_value = True
+    print(has_value)
```

Comment:
```suggestion
def check_lunchbox(items: list[str]):
    empty = False
    if not items:
        empty = True
    print(empty)
```

```suggestion
def check_lunchbox(items: list[str]):
    empty = False if items else True
    print(empty)
```

```suggestion
def check_lunchbox(items: list[str]):
    empty = bool(not items)
    print(empty)
```

Might be three possibility each in their own directory ? (Not actually doing the change before I have your opinion about it though).

Reply:
I find the examples with the ternaries a little distracting. The bool is nice, but also a little distracting since it won't work in every case.